sdk/core/core/src/az_span.c

```diff
@@ -226,74 +211,55 @@ AZ_INLINE void _az_uint8_swap(uint8_t* a, uint8_t* b)
 }
 
 /**
- * @brief move the content from span @b a to span @b b and viceverse.
+ * @brief move the content from span @span1 to span @span2 and viceverse.
```

Comment:
```suggestion
 * @brief move the content from span @span1 to span @span2 and vice versa.
```

Reply:
Also, should this documentation live on the `az_span_private.h` header file instead of the source?

Reply:
Yes. We should document the internal headers for our partners/client library engineers. This is just lower priority